warehouse/templates/packaging/detail.html

```diff
@@ -119,7 +123,11 @@ <h1 class="package-header__name">
     </div>
 
     <div class="package-header__right">
-    {% if release.yanked %}
+    {% if release.is_draft %}
+      <a class="status-badge status-badge--warn" href="{{ request.route_path('packaging.project', name=release.project.name) }}">
+        <span>{% trans %}Draft version{% endtrans %}</span>
```

Comment:
This doesn't release the version, it takes you to the default (latest) version of the project :sweat_smile:  (similar to when you look at an old release)

But adding such a button would be nice :thinking: the only problem is that this page can be viewed by any user with the right link, and not only maintainers (that's why the publish button is under `Manage project`)

Reply:
If the viewer is not a maintainer, what would happen when they click on the button? If it can emit a nice-ish error, having it here would be a good idea. I’m assuming a viewer will hit a 403 at some point if they don’t have publishing permissions right now.